opencga-catalog/src/main/java/org/opencb/opencga/catalog/managers/OrganizationManager.java

```diff
@@ -357,4 +364,19 @@ public List<String> getOrganizationIds(String token) throws CatalogException {
         authorizationManager.checkIsOpencgaAdministrator(tokenPayload, "get all organization ids");
         return catalogDBAdaptorFactory.getOrganizationIds();
     }
+
+    private void privatizeResults(OpenCGAResult<Organization> result) {
```

Comment:
Also known as "redact"
https://www.altlaw.co.uk/blog/data-redaction-vs-data-masking-the-key-difference